docs:faq for anti (#788)
taptap/tds-docs: i18n/en/docusaurus-plugin-content-docs/current/sdk/anti-addiction/faq.mdx

### In a testing environment for real-name verification, the verification process is still required for test accounts

Please make sure that the **player's unique identifier** `userIdentifier` passed into `startup` is the `unionid` of the TapTap user, which can be obtained by referring to [Quick Log-in With TapTap](/sdk/taptap-login/guide/start/#quick-log-in-with-taptap).

> ### Use TapTap Quick Authentication: Could not get real name information, please try again later.

Reviewer: ```suggestion
### I see the error "Could not get real name information, please try again later" when using TapTap Quick Authentication
```

> - As of v3.22.0, TapTap Quick Authentication will not report this exception if the account logged in by the TapTap client is not a real name in the TapTap client, but will jump to the TapTap client for real name authentication.

Reviewer: ```suggestion
- Verify that the account logged in to the TapTap client has completed real-name verification in the client. As of v3.22.0, TapTap Quick Authentication will not throw an exception if the account logged in to the TapTap client has not completed real-name verification in the TapTap client, but will take the user to the TapTap client for real-name verification.
```

> - Check that the device time is synchronised with the network, an inaccurate device time will also cause this exception.

Reviewer: ```suggestion
- Check that the device time is synchronized with the network. An inaccurate device time will also cause this exception.
```
